app/Jobs/RequestProjectCheckUrl.php

```diff
@@ -48,13 +49,24 @@ public function handle(Client $client)
             $link->last_log = null;
 
             try {
-                $client->get($link->url);
+                $request = $client->get($link->url);
+
+                if (!empty($link->match) && !is_null($link->match)) {
+                    $body = (string)$request->getBody();
+                    if (! preg_match('~'.htmlentities(trim($link->match)).'~i', $body)) {
```

Comment:
You need to use `preg_quote` when using user input in a regular expression

Reply:
Fixed, thanks